pkg/operator/operator/validations.go

```diff
@@ -446,12 +470,25 @@ func validatePredictor(predictor *userconfig.Predictor, projectFileMap map[strin
 		if err := validateTensorFlowPredictor(predictor); err != nil {
 			return err
 		}
+		if err := validateDockerImagePath(predictor.TFServeImage); err != nil {
+			if errCause := errors.Cause(err); errCause != nil {
+				return errors.Wrap(errCause, userconfig.TFServeImageKey, err.Error())
```

Comment:
I believe the original error message should be included in `err` without the wrap being necessary, or was there a different reason you extracted the cause here? (also below)

Reply:
Modified it as you have suggested it in the other comment (aka [example](https://github.com/cortexlabs/cortex/blob/master/pkg/lib/aws/errors.go#L70)).